panimg/image_builders/tiff.py

```diff
@@ -18,15 +18,18 @@
 
 import tifffile
 
-from panimg.contrib.wsi_dcm_to_tiff.dcm_to_tiff import (
-    dcm_to_tiff as wsi_dcm_to_tiff,
-)
-from panimg.contrib.wsi_dcm_to_tiff.dcm_to_tiff import is_dicom as is_wsi_dicom
 from panimg.exceptions import UnconsumedFilesException, ValidationError
+from panimg.image_builders.dicom import get_dicom_headers_by_study
 from panimg.models import MAXIMUM_SEGMENTS_LENGTH, ColorSpace, TIFFImage
 
 try:
-    import openslide
+    if hasattr(os, "add_dll_directory"):
+        # Windows
+        OPENSLIDE_PATH = r"C:\Program Files\OpenSlide\bin"
+        with os.add_dll_directory(OPENSLIDE_PATH):
+            import openslide
+    else:
+        import openslide
```

Comment:
Wow, this looks pretty random... why is this needed, or better said: Who needs this? Normally you should just setup your runtime environment correctly. This looks like very wild, random path guessing to me...

Reply:
For the record, with "Setup your runtime" I mean:

~~~~~.bat
set PATH=%PATH%;C:\Program Files\OpenSlide\bin
python3 do-what-I-want.py
~~~~~

Reply:
that wasn't supposed to be committed...

Reply:
:rofl: - ok, well spotted by me then... neverthelss, I _can_ see the utility. Maybe we want to add it to the docs somehow? We could _also_ provide a snippet like this to run as part of:

https://docs.python.org/3/using/cmdline.html?highlight=pythonpath#envvar-PYTHONSTARTUP

just some suggestions!